rules/network/initial_access_rdp_remote_desktop_protocol_to_the_internet.toml

```
@@ -32,7 +32,7 @@ tags = ["Elastic", "Network"]
 type = "query"
 
 query = '''
-network.transport:tcp and destination.port:3389 and
+event.category:(network or network_traffic) and network.transport:tcp and (destination.port:3389 or event.dataset:zeek.rdp or (event.dataset:zeek.ssl and destination.port:3389)) and
```

```suggestion
event.category:(network or network_traffic) and network.transport:tcp and (destination.port:3389 or event.dataset:zeek.rdp) and
```
redundant (similar to above)